### DISCUSSION TITLE
Write out the basic formatting on the text-note page

### DISCUSSION THREAD

**Discussion by LilMouse** [TriliumNext/Trilium#4068] [Ideas]
On the Text note wiki page I found a gif which shows the basic formatting.
It also links to the the CKEditor 5 docs for their formatting.
But maybe we can add a small overview to the note wiki page too.

I was thinking something like following. It shows the basics

## Basic Format
**Bold** – Type `**text**` or `__text__`,
*Italic* – Type `*text*` or `_text_`,
`Code` – Type \`text\` ,
~~Strikethrough~~ – Type `~~text~~`.

## Lists
[Bulleted list](https://ckeditor.com/docs/ckeditor5/latest/features/lists/lists.html) – Start a line with * or - followed by a space.
[Numbered list](https://ckeditor.com/docs/ckeditor5/latest/features/lists/lists.html) – Start a line with 1. or 1) followed by a space.
[To-do list](https://ckeditor.com/docs/ckeditor5/latest/features/lists/todo-lists.html) – Start a line with [ ] or [x] followed by a space to insert an unchecked or checked list item, respectively.
Blocks

## Blocks
[Block quote](https://ckeditor.com/docs/ckeditor5/latest/features/block-quote.html) – Start a line with > followed by a space.
[Code block](https://ckeditor.com/docs/ckeditor5/latest/features/code-blocks.html) – Start a line with ```.

## Other
[Headings](https://ckeditor.com/docs/ckeditor5/latest/features/headings.html) – Start a line with ## or ### followed by a space to create a heading 1, heading 2, or heading 3 (up to heading 6 if [options](https://ckeditor.com/docs/ckeditor5/latest/api/module_heading_headingconfig-HeadingConfig.html#member-options) defines more headings).
Note: Trilium only accepts headings with ## and more because # is reserved for the title.
[Horizontal line](https://ckeditor.com/docs/ckeditor5/latest/features/horizontal-line.html) – Start a line with ---.

Note: I couldn't get the "`" character inside a code block so I would like to know if that is possible.

**Comment by XXXJetfireXXX** [comment]
One small correction: Trilium only uses headings from H2 (`##`) and so on. #1552 explains why.

**Reply by LilMouse** [reply]
Good point! I copied it from CKEditor documentation. Will edit my original post.

**Comment by meichthys** [comment]
Back tick character works in code blocks:
![image](https://github.com/zadam/trilium/assets/10717998/e173c1b2-c078-4b2f-abcd-f4ec8a8ebbed)
create the code block first, then enter the back tick

**Comment by meichthys** [comment]
This has been  added to the wiki.
